content/en/docs/tasks/administer-cluster/dns-debugging-resolution.md

```diff
@@ -247,6 +251,20 @@ linux/amd64, go1.10.3, 2e322f6
 172.17.0.18:41675 - [07/Sep/2018:15:29:11 +0000] 59925 "A IN kubernetes.default.svc.cluster.local. udp 54 false 512" NOERROR qr,aa,rd,ra 106 0.000066649s
 ```
 
+### Are you in the right namespace for the service?
+
+DNS queries for services change based on namespace of the pod. DNS queries that don't specifiy a namespace are limited to the pod's namespace. 
```

Comment:
why are we repeating these two sentences here?

Reply:
I think the step "are you in the right namespace for the service?" should be included in debugging DNS resolution. The other sentences are merely supporting information.